addon/components/nypr-m-box-banner.js

```diff
@@ -0,0 +1,28 @@
+// BEGIN-SNIPPET nypr-m-box-banner.js
+import Component from '@ember/component';
+import layout from '../templates/components/nypr-m-box-banner';
+
+/**
+  A box for rendering arbitrary content into `title` and `body` areas.
+
+  Will render a `close` button if a `@close` argument is passed.
+
+  @class nypr-m-box-banner
+  @yield {Block} block
+  @yield {Component} block.title `nypr-m-box-banner/title`
+  @yield {Component} block.dek `nypr-m-box-banner/dek`
```

Comment:
I am maybe not newsy enough but I didn't know what dek was (I figured it out based on context, but just remember that dummies like me are gonna be using this)

Reply:
yeah the `dek` is something that's usd all over pattern lab, so I decided to give in and start using it too